packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1046,6 +1046,11 @@ describe("SharedTree", () => {
 
 			// ensure the remove is out of the collab window
 			assert(removeSequenceNumber < provider.minimumSequenceNumber);
+			// check that the repair data on the peer is destroyed
+			const repairCursor2 = tree2.checkout.forest.allocateCursor();
+			moveToDetachedField(tree2.checkout.forest, repairCursor2, brand("repair-4"));
```

Comment:
Using "repair-4" seems really brittle because it's an implementation detail. I think it would be better to create an anchor to the node of interest (before that node is removed) then check that the node is destroyed by checking that attempting to locate the anchor return undefined.

Reply:
This comment was marked as resolved but the code doesn't seem to have changed. Am I missing something?

Reply:
Resolving now that I have addressed it.